src/test/java/org/junitpioneer/jupiter/IssueTests.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.junitpioneer.testkit.ExecutionResults;
+import org.junitpioneer.testkit.PioneerTestKit;
+
+public class IssueTests {
+
+	@Issue("bishue 135")
+	@Test
+	void writeToReport() {
+		// This tests should get covered by the registered PioneerReportListener.
+		// In the report should then be an property with key "issue" and value "bishue 135" for this test
+
+		/*
+		    <testcase name="writeToReport()" status="SUCCESSFUL">
+		        <properties>
+		            <property name="Issue" value="bishue 135"/>
+		        </properties>
+		    </testcase>
+		 */
+
+		// The report can't be accessed within this test as the file is written after this (and all others)
+		// tests are finished
+		assertThat(true).isTrue();
+	}
+
+	@Test
+	void checkMethodNotAnnotated() {
```

Comment:
This makes it sound like you check if the method is annotated or not. I think this should be called `checkNotAnnotatedMethod` and have a `@DisplayName` for clarity's sake.

Reply:
Will rename it and add a `@DisplayName`